tests/db_access/test_utils.hpp

```diff
@@ -7,7 +7,34 @@
 
 #include <chrono>
 #include <thread>
+#include <iostream>
+#include <streambuf>
+#include <string>
+#include <sstream>
 
+template <typename T>
+class StreamRedirect {
+    public:
+        StreamRedirect(T &stream): buffer{}, originalStream{stream},
```

Comment:
neni ten buffer{} zbytecny?

Reply:
Jsem si jisty ze musi byt inicializovany pred pouzitim, nevim jak presne se kompilator zachova kdyz ho nezinicializuju explicitne, IMO "explicit is better than implicit"

Reply:
Considering that buffer is actually required within the constructor, I agree here with explicitness.